src/components/Data/Documents/Views/Column/Column.vue

```diff
@@ -353,23 +353,43 @@ export default {
       }
       return {
         label: getBadgeText(n.action),
-        variant: getBadgeVariant(n.action)
+        variant: getBadgeVariant(n.action),
       }
     },
     promptExportCSV() {
       this.$bvModal.show(this.csvExportPromptModalID)
     },
     exportToCSV() {
-      const blob = new Blob([
-        convertToCSV(this.formattedItems, this.selectedFields)
-      ])
-      const a = document.createElement('a')
-      a.download = `${this.index}-${this.collection}.csv`
-      a.href = URL.createObjectURL(blob)
-      a.addEventListener('click', () => {
-        setTimeout(() => URL.revokeObjectURL(a.href), 30 * 1000)
+
+      // Request to the export api endopoint
+      let request = new XMLHttpRequest()
+      request.open(
+        'POST',
+        `http://localhost:7512/${this.index}/${this.collection}/_export?format=csv`
+      )
+      request.setRequestHeader('Content-Type', 'application/json')
+      request.responseType = 'text'
+
+      // Request body with the selected fields and the current querry
+      let body = JSON.stringify({
```

Comment:
Maybe we should put this HTTP call in the store (since it's a backend interaction) to access the SDK instance and reuse values from it:

This is only an example not sure the name of the variables:
```suggestion
      // Request to the export api endopoint
      let request = new XMLHttpRequest()
      request.open(
        'POST',
        `${this.sdk.protocol.ssl ? 'https' : 'http'}://{this.sdk.protocol.host}:${this.sdk,protocol.port}/${this.index}/${this.collection}/_export?format=csv`
      )
      request.setRequestHeader('Content-Type', 'application/json')
      request.setRequestHeader('Authorization', `Bearer ${this.sdk.jwt}`)
      request.responseType = 'text'

      // Request body with the selected fields and the current querry
      let body = JSON.stringify({
```

What do you think @Aschen @Shiranuit @sebtiz13?

Reply:
I think it's not really useful to move this on the store, because this action doesn't need to save the result.
IMHO, the store should be seen like a local storage for browser, to store data we need to share between different pages and components.

Reply:
I agree with you about the move the Vue.js store. But what do you think about reuse all the SDK info (including the current JWT)?

Reply:
I think it brings a solution on the discussion of @Shiranuit  and @Aschen  about the need to pass the token in the URL

Reply:
This PR is stil WIP. The code you are talking about has been removed anyway